generators/client/templates/react/src/main/webapp/app/modules/administration/tracker/tracker.tsx.ejs

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Translate } from 'react-jhipster';
+import { connect } from 'react-redux';
+
+export interface ITrackerPageProps {
+  activities: any[];
+}
+
+export class TrackerPage extends React.Component<ITrackerPageProps> {
```

Comment:
the component doesnt have state or use lifecycle methods, convert to a arrow method
```
export const TrackerPage = ({ activities }:ITrackerPageProps) => (
   <div>
   ...
   </div>
);

const mapStateToProps = ({ administration }) => ({
  activities: administration.tracker.activities
});

export default connect(mapStateToProps)(TrackerPage);
```